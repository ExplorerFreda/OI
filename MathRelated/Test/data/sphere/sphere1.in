3
19925.000000 1996.500000 13958.500000
7301.500000 2692.500000 5560.500000
15704.000000 13620.500000 1241.000000
6272.000000 18605.000000 11545.500000
