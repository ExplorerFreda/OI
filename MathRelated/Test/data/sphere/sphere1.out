13062.923 10049.695 11225.310
